app/navigation/ReturnsJourneyNavigator.scala

```diff
@@ -158,15 +163,38 @@ class ReturnsJourneyNavigator @Inject()(
   private def directlyExportedComponentsRoute(answers: UserAnswers, mode: Mode): Call =
     answers.get(DirectlyExportedComponentsPage) match {
       case Some(true) => routes.ExportedPlasticPackagingWeightController.onPageLoad(mode)
-      case Some(false) => routes.NonExportedHumanMedicinesPlasticPackagingController.onPageLoad(mode)
+      case Some(false) => routes.PlasticExportedByAnotherBusinessController.onPageLoad(mode)
       case _ => throw new Exception("Unable to navigate to page")
     }
 
   private def exportedPlasticPackagingWeightRoute(answers: UserAnswers, mode: Mode): Call =
-    exportedAllPlastic(answers) match {
-      case true => routes.ReturnsCheckYourAnswersController.onPageLoad()
-      case false => routes.NonExportedHumanMedicinesPlasticPackagingController.onPageLoad(mode)
+    if(mode.equals(CheckMode)) {routes.PlasticExportedByAnotherBusinessController.onPageLoad(mode)}
+    else {
+      exportedAllPlastic(answers) match {
+        case true => routes.ReturnsCheckYourAnswersController.onPageLoad()
+        case false => routes.PlasticExportedByAnotherBusinessController.onPageLoad(mode)
+      }
```

Comment:
//todo in another pr if you want, but can we move these away from calling the general navigator

Reply:
happened in 447

Reply:
yes, I did that in 447 as this was only to created the most of the skelaton